spec/jobs/user_cleanup_job_spec.rb

```diff
@@ -35,8 +35,7 @@
 
     context 'when with will archive and archived users' do
       it { expect(ActionMailer::Base.deliveries.count).to eq 1 }
-      it { expect(email.to).to include 'bestuur@csvalpha.nl' }
-      it { expect(email.to).to include 'ict@csvalpha.nl' }
+      it { expect(email.to).to include Rails.application.config.x.privacy_email }
```

Comment:
Again, this is not just a refactor, you're modifying the values. Is that intentional? It wasn't in the issue description

Reply:
this is the test for the file above so it should also be changed

Reply:
I edited the pr description to better reflect the changes